src/imagecapturecore.cs

```diff
@@ -456,9 +456,15 @@ interface ICCameraDeviceDelegate : ICDeviceDelegate {
 		void DidChangeCapability (ICCameraDevice camera);
 
 		[Abstract]
+		[Deprecated (PlatformName.MacOSX, 10, 15, message: "Use 'DidReceiveThumbnailForItem (ICCameraDevice, CGImageRef, ICCameraItem, NSError)' instead.")]
+		[Deprecated (PlatformName.MacCatalyst, 13, 1, message: "Use 'DidReceiveThumbnailForItem (ICCameraDevice, CGImageRef, ICCameraItem, NSError)' instead.")]
 		[Export ("cameraDevice:didReceiveThumbnailForItem:")]
 		void DidReceiveThumbnail (ICCameraDevice camera, ICCameraItem forItem);
 
+		[Abstract]
+		[Export ("cameraDevice:didReceiveThumbnail:forItem:error:")]
+		void DidReceiveThumbnailForItem (ICCameraDevice camera, /* CGImageRef */ IntPtr thumbnail, ICCameraItem forItem, [NullAllowed] NSError error);
```

Comment:
Can't we do CGImage here? 

```suggestion
		void DidReceiveThumbnailForItem (ICCameraDevice camera, CGImage thumbnail, ICCameraItem forItem, [NullAllowed] NSError error);
```

Reply:
@dalexsoto I thought about that, but after reading [this](https://stackoverflow.com/questions/30909126/why-use-cgimageref-not-cgimage-instead) and looking at how we handled CGImageRef in other parts of the codebase, I thought this implementation was better. But if the overall consensus is that using CGImage is a better option, I can change it.

Reply:
@dalexsoto we are dealing with a delegate in this case.

Reply:
Yeah also `CGImageRef` and `CGImage` seem not to be toll-free bridged so this is ok!